apps/website/src/pages/docs/header.mdx

```diff
@@ -14,58 +14,39 @@ import * as AllExamples from 'examples';
 
 <p>{frontmatter.description}</p>
 
-<Placeholder componentPageName='core-header' />
-
 <LiveExample src='Header.main.tsx'>
   <AllExamples.HeaderMainExample client:load />
 </LiveExample>
 
-The headers seen when in our CONNECT Portal environment. A header sits at the top of a page and is part of a consistent user experience.
-
-## Main CONNECT Portal header
+The first thing most people look at when they arrive at a new web page is the header. It is usually stuck at the top of the page. The header contains vital information about your company's name and function, as well as the purpose of the specific page.
 
-The CONNECT portal header contains the services menu, home, enterprise, assets and projects links beginning from left to right. From right to left there is the user profile, help, and notification links.
+## Customizations
 
-Header with circular callouts, numbered from 1 to 8.
+Alter standard header section by adding new custom elements and content.
 
-- Services menu
-- Home page
-- Enterprise portal
-- Assets menu
-- Projects menu
-- Notifications
-- Help
-- User icon
+### Header with MenuItems
 
-### Actions
+Have default header with menu items using `breadcrumbs` prop.
 
-- Services menu pops up a list of services available to the user.
-- Home page takes you to CONNECT Center.
-- Enterprise Portal takes you to the Subscription Services Portal.
-- Assets menu takes you to the assets page.
-- Projects menu takes you to the ProjectWise Projects page.
-- Notifications pops up your list of notifications.
-- Help takes you to Bentley Communities for Bentley Cloud and Web Services.
-- User profile is a drop down with options for your user profile.
+<LiveExample src='Header.menuitems.tsx'>
+  <AllExamples.HeaderFullExample client:load />
+</LiveExample>
 
-## Secondary header
+### Slim Header
 
-The secondary header contains the product icon, product name, iModel name and the name of the initially opened view. On the right side of the bar is the settings menu.
+Make header slim using `isSlim` property.
 
-Secondary header with circular callouts, numbered from 1 to 5.
+<LiveExample src='Header.slim.tsx'>
+  <AllExamples.HeaderSlimExample client:load />
+</LiveExample>
 
-- Product icon
-- Product name
-- iModel name
-- View name
-- Settings
+### Center Content
```

Comment:
lets remove this center content section and example, as it's not part of the component.